examples/complete/main.tf

```diff
@@ -20,5 +20,6 @@ resource "aws_db_instance" "default" {
 module "rds_alarms" {
   source         = "../../"
   db_instance_id = aws_db_instance.default.id
+  depends_on     = [aws_db_instance.default]
```

Comment:
This line is not necessary. Because line 22 references the aws_db_instance.default resource, the dependency is implicit

Reply:
And yet... one of the tests was failing because the instance wasn't created before the test had a chance to complete (see #56 and [my comment](https://github.com/cloudposse/terraform-aws-rds-cloudwatch-sns-alarms/pull/56#issuecomment-1987220991)). 

I tried (obviously wrong) to fix and help with merging this (minor, but ridiculously complicated to merge) change.

I agree it makes no sense to keep `depends_on` there. Please let me know if you want me to remove this line from code and make another commit.

Thank you!

Reply:
OK, unnecessary line deleted.